parquet-column/src/main/java/org/apache/parquet/column/statistics/BinaryStatistics.java

```diff
@@ -53,12 +53,12 @@ public void setMinMaxFromBytes(byte[] minBytes, byte[] maxBytes) {
 
   @Override
   public byte[] getMaxBytes() {
-    return max.getBytes();
+    return hasNonNullValue() ? max.getBytes() : null;
```

Comment:
This gets called from within Jackson when converting to JSON, so we don't have much of a choice.